admin-tools/lambda/src/main/scala/com/gu/mediaservice/BatchIndexLocalHandler.scala

```diff
@@ -0,0 +1,36 @@
+package com.gu.mediaservice
+
+import com.amazonaws.auth.profile.ProfileCredentialsProvider
+
+import scala.concurrent.ExecutionContext.Implicits.global
+
+object BatchIndexLocalHandler extends App {
+
+  lazy val awsCredentials = new ProfileCredentialsProvider("media-service")
+
+  private val cfg = BatchIndexHandlerConfig(
+    apiKey = "dev-",
```

Comment:
Can this be pushed to config rather than hard coded?

Reply:
this is BatchIndexLocalHandler for local run only like for dev i will move that to dev pakcage

Reply:
it is the same approach as we sued for projection api inside play

Reply:
done